### DISCUSSION TITLE
关于使用文件列表如何变为纯内网版？

### DISCUSSION THREAD

**Discussion by LiJunXiao** [AlistGo/alist#610] [Q&A]
现在需求是，把服务器的某个文件夹共享出来，用于提供给同路由器下连接的设备访问服务器的文件。所以阿里云盘、百度云盘、等并不需要。只需要本地存储。同时服务器是一个纯内网的服务器，如何能把项目中本身相关的cdn的js文件等换成本地文件。大佬能解答一下嘛，呜呜呜。

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/setting/config#assets

**Reply by LiJunXiao** [reply]
大佬，去您发的那个网页链接看了一下，请问这个config.json 在哪里设置呢。我自己本地是下载了windows版本的压缩包，只有一个.exe文件，运行起来后，只看到前端设置和后端设置，并没有看到config.json文件。请问是通过什么命令行设置么？

**Reply by xhofe** [reply]
运行会自动生成 在程序所在目录的data/config.json

**Comment by faith01238** [comment]
请问你解决这个问题了吗？我看不太懂这个说明，配置文件没有找到相关字段
